app/models/miq_server/worker_management/process.rb

```diff
@@ -1,4 +1,10 @@
 class MiqServer::WorkerManagement::Process < MiqServer::WorkerManagement
+  attr_accessor :miq_processes
+
+  def sync_from_system
+    self.miq_processes = miq_workers.pluck(:pid).map { |pid| MiqProcess.processInfo(pid) }
```

Comment:
Technically these are children of the orchestrator process right?  Wonder if it's better to query the system directly (unless you are going for parity first)

Reply:
Yeah that's true, I'll see if I can query by `ppid`

Reply:
:+1: done that came out nicely thanks